x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/utils/validate.ts

```diff
@@ -128,3 +130,25 @@ function ruleObjectContainsResponseActions(
 ): rule is Rule<UnifiedQueryRuleParams | EsqlRuleParams | EqlRuleParams | NewTermsRuleParams> {
   return rule != null && 'params' in rule && 'responseActions' in rule?.params;
 }
+
+export const validateNonCustomizableUpdateFields = (
+  ruleUpdate: RuleUpdateProps,
+  existingRule: RuleResponse
+) => {
+  if (!isEqual(ruleUpdate.author, existingRule.author)) {
+    throw new ClientError(`Cannot update "author" field for prebuilt rules`, 400);
+  } else if (ruleUpdate.license !== existingRule.license) {
+    throw new ClientError(`Cannot update "license" field for prebuilt rules`, 400);
+  }
```

Comment:
Just to confirm `update` logic here, we want to throw on unsetting fields here too, correct? This is the only difference between the update and patch validation methods in this file

Reply:
Yes, that's correct. No changes should be allowed to the fields, including unsetting them.